### PR TITLE
Feat: Improved the popup marker with vital informations like the repo…

### LINKED ISSUE #97: [FEATURE] Popup Improvement
### Description

This issue is to review and add useful information to the map markers

Current map marker popup:
screenshot of a round image of amanda marking. location on leaflet map with a popup open

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ScottDeev**
… in which the activity was carried out and also some other informations

## Fixes Issue #97 

Closes #97 

## Changes proposed

Added vital information to the marker pop up on the map according to the information contained on the event endpoint

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
Screenshot (346)

## Note to reviewers
More suggestions on the information to be added to the marker can be made

### CODE REVIEW

File: src/components/map.jsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import PanToMarker from "./zoom-to-marker";
 import PropTypes from "prop-types";
 
 function Map({ events }) {
+  console.log(events);
```
Comment (amandamartin-dev) [inline src/components/map.jsx:11]: Don't forget to remove the console log

File: src/components/map.jsx [hunk 2]
```diff
@@ -34,7 +35,13 @@ function Map({ events }) {
             position={[event.githubUsername.location.lat, event.githubUsername.location.long]}
           >
             <Popup>
-              {event.githubUsername._id}: {event.event}
+              <div className="flex flex-col gap-[5px]">
+                <h1 className="font-[600]">{event.githubUsername._id}</h1>
+                <span>Location - {event.githubUsername.location.provided}</span>
+                <span>Repo - {event.repoName}</span>
```
Comment (amandamartin-dev) [inline src/components/map.jsx:41]: For the map, you only get one point added no matter how many total events.  Does it update to the most recent event each time? Or is it showing the first event the user added at the start or the socket stream?  Should this be clear in the popup? wdyt?

Comment (ScottDeev) [reply src/components/map.jsx:41]: It updates to the most recent event. Having more than one point from the same location on the map will crowd it up

File: src/components/map.jsx (hunk 1 above)
Comment (ScottDeev) [reply src/components/map.jsx]: Fixed

File: src/components/map.jsx (hunk 2 above)
Comment (amandamartin-dev) [reply src/components/map.jsx:40]: Oh yes we definitely don't want additional points, just was askign about the popup text. That's perfect. Thank you!

**Review by amandamartin-dev — APPROVED** [review]
LGTM! Thanks!

File: src/components/map.jsx (hunk 2 above)
Comment (ScottDeev) [reply src/components/map.jsx:40]: Oh. Alright. Thank you

**Pull request merged**